### PR TITLE
env-validation

### PR DESCRIPTION
**Pull request by bolubee101**
### env-validation
Updated env validation with the following
- REDIS_URL: string
- MONGODB_URL: string
- JWT_SECRET: string

### CODE REVIEW

**Comment by Emmanuerl** [comment]
> ### env-validation
> Updated env validation with the following
> 
> * REDIS_URL: string
> * MONGODB_URL: string
> * JWT_SECRET: string

@bolubee101 we're currently trying to achieve https://docs.nestjs.com/techniques/configuration . We've setup the schema, quite alright, what's remaining is for us to configure the ConfigModule and make the ConfigService injectable

Also, you might also want to add an environment map somewhere so we don't have things like
 ```javascript
this.configService.get("JWT_SECRET");
``` 
but rather 
```typescript
this.configService.get(keys.jwt.secret);
```

**Review by Emmanuerl — CHANGES_REQUESTED** [review]
https://github.com/Q-Devs-ng/bartimeus/pull/3#issuecomment-1193370530

File: src/app.module.ts [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+import { ConfigModule } from '@nestjs/config';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { validate } from './config/env';
 
 @Module({
-  imports: [],
+  imports: [
+    ConfigModule.forRoot({
+      envFilePath: '.development.env',
```
Comment (Emmanuerl) [inline src/app.module.ts]: why is this .development.env ?

**Review by Emmanuerl — CHANGES_REQUESTED** [review]
I also can't see the URI scheme validation for mongodb and redis

File: src/app.module.ts (hunk 1 above)
Comment (bolubee101) [reply src/app.module.ts]: I would update it.

**Review by Emmanuerl — APPROVED** [review]

**Pull request merged**